server/app.py

```
@@ -0,0 +1,230 @@
+@app.post("/chat")
+def chat(request: MessageRequest):
+    tree = load_tree(request.conversation_id)
+    try:
+        response = tree.chat(request.message)
+        save_tree(tree) # Note: save_tree relies on global current_file_id which might be wrong!
+        # Fix save_tree to take path or ID too?
+        # Let's fix save_tree uses.
+        path = get_file_path(request.conversation_id)
+        tree.save_to_file(path)
+        
+        return {"response": response}
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)